src/containers/CourseListContainer/CourseListContainer.js

```diff
@@ -13,11 +14,14 @@ import { STACK_SCREENS } from '../CTNavigationContainer/index'
  */
 const CourseListContainer = ({ departmentId, acronym, navigation }) => {
   const [courses, setCourses] = useState([])
+  const setLoading = useLoadingIndicator()
 
   useEffect(() => {
     const fetchCourseInfo = async () => {
+      setLoading(true)
```

Comment:
```javascript
setLoading(true)
...
setLoading(false)
```
seems to be a common pattern. Could it be improved by introducing a new wrapper function?
For example,
```javascript
const wrapper = async (asyncFun) => {
    setLoading(true)
    await aysncFun()
    setLoading(false)
}
```